HTML+CSS+JS/Forms/Glassmorphism-Form/README.md

```diff
@@ -0,0 +1,13 @@
+## Glassmorphism Animation Login Form.
+
```

Comment:
Mention some info about it and if the form is responsive or not so that people can decide if they want to use this form in their application.